docs/data/sources/shopify.md

```diff
@@ -5,66 +5,39 @@ description: Amplitude’s app with Shopify is a smart analytics app that automa
 
 Amplitude’s app with Shopify is a smart analytics app that automates eCommerce tracking. With Amplitude’s app, you can automatically track and ingest key e-commerce events, event properties, and user properties on a Shopify or Shopify Plus store.
 
-## Considerations
+### Client-side (device mode) tracking
+During the installation process, Amplitude Shopify App adds the code snippet to all pages (included in theme.liquid) on your Shopify store. The benefits of this approach include:
+- Amplitude’s JS SDK library is loaded on all pages, except for the checkout
+
+### Server-side (cloud mode) tracking
+During the Amplitude connection setup, our Shopify App also adds a set of webhooks to your Shopify store. When a customer interacts with your store these changes are relayed server-side from Shopify to Amplitude. The advantages to this approach are:
+- 100% event capture for adds to cart, checkout steps, sales.
+- Customer data (for example, email) securely relayed server-side.
+- No extra scripts on the sensitive and secure checkout pages.
+- Accurate marketing attribution, even when customers use ad-blockers or cookie opt-outs.
+- More reliable and trustyworthy than client-side events (as it represents final, concluded transactions or states within Shopify).
 
-- The Shopify integration is available for all Amplitude plans.
-- The Shopify App was built to solve the general use cases for most Shopify stores. We designed a generalized Out of the Box Taxonomy to include events and properties that most Shopify stores would be interested in tracking.
+## Considerations
+- The Amplitude app is free to install.
+- You will need either an Amplitude Starter, Growth, or Enterprise plan to start sending data from your Shopify store.
+- The Shopify App was built to solve the general use cases for most Shopify stores. We've designed a generalized Out of the Box Taxonomy to include events and properties that most Shopify stores would be interested in tracking. 
 - We recommend leveraging our Amplitude SDKs or APIs in addition to this app if your Shopify Store has a high degree of in-built functionality or if you require instrumenting a lot of custom events outside our taxonomy list. 
+- All events sent by the Shopify app have the preheader `[Shopify]` to distinguish themselves from the custom events.
```

Comment:
```suggestion
- All events sent by the Shopify app have the preheader `[Shopify]` to help distinguish them from custom events.
```

Reply:
I think we typically refer to these "preheaders" like "the events are prefixed with ..." -- would prefer to be consistent here if it's accurate.